packages/admin-next/dashboard/src/hooks/api/fulfillment.tsx

```diff
@@ -35,7 +35,7 @@ export const useCancelFulfillment = (
     onSuccess: (data: any, variables: any, context: any) => {
       queryClient.invalidateQueries({ queryKey: fulfillmentsQueryKeys.lists() })
       queryClient.invalidateQueries({
-        queryKey: ordersQueryKeys.details(),
+        queryKey: ordersQueryKeys.all,
```

Comment:
There is no need to invalidate everything on `orders` if we just want to refresh preview.
You can pass `orderId` as a param in the hook and call `ordersQueryKeys.preview(id)`.

Reply:
good push, I'll do this for all the other ones i've done so far as well.

Reply:
@fPolic just changed. Invalidating just the `.detail(id)` isn't invalidating it correctly though.